lib/breakers/statsd_plugin.rb

```diff
@@ -46,5 +31,30 @@ def send_metric(status, service, request_env, response_env)
         StatsD.measure(metric_base + 'time', response_env[:duration], tags: tags)
       end
     end
+
+    private
+
+    def filtered_endpoint_tag(path)
+      # replace identifiers with 'xxx'
+      # this nasty-looking regex attempts to cover:
+      # * (possibly negative) digit identifiers
+      # * uuid's with or without dashes
+      # * institution id's of form 111A2222 or 11A22222
+      digit = /\-?\d+/
+      contact_id = /\d{10}V\d{6}(%5ENI%5E200M%5EUSVHA)*/
+      uuids = /[a-fA-F0-9]{8}(\-?[a-fA-F0-9]{4}){3}\-?[a-fA-F0-9]{12}/
+      institution_ids = /[\dA-Z]{8}/
+      provider_ids = /Providers\(\d{10}\)/
+      v1_user_ids_1 = /[0-9a-f]{22}$/ # e.g. /api/v1/users/bb9c0f499977be68611151
+      v1_user_ids_2 = /00u2[0-9a-zA-Z]{16}/ # e.g. /api/v1/users/00u2gskfa6kXUvU0N792
+
+      r = %r{
+        (\/)
+        (#{digit}|#{contact_id}|#{uuids}|#{institution_ids}|#{provider_ids}|#{v1_user_ids_1}|#{v1_user_ids_2})
+        (\/|$)
+      }x
```

Comment:
Had to refactor this regex to be multi-line because adding `|#{v1_user_ids_1}|#{v1_user_ids_2}` broke the rubocop line length.

Reply:
I could use a 2nd pair of eyes... are these two equivelant?
```ruby
r = %r{(\/)(/foo/|/bar/})(\/|$)}
```
I changed it to:
```ruby
r = %r{
  (\/)
  (/foo/|/bar/)
  (\/|$)
}x
```

Reply:
why not just leave it on one line and add to the rubocop ignore?

Reply:
I suppose you could make the case for ignore since fixing could introduce a bug, but we do have a pretty good spec that makes me confident that I didn't. This line is pretty unreadable, so I think a fix is warranted.